libmscore/scorefile.cpp

```diff
@@ -449,10 +449,15 @@ bool Score::saveFile()
       QFile::setPermissions(name, QFile::ReadOwner | QFile::WriteOwner | QFile::ReadUser
          | QFile::ReadGroup | QFile::ReadOther);
 
+      startCmd();
+      info.refresh();         // update file info
+      setLayoutAll(true);
+      endCmd();               // force relayout
```

Comment:
Pretty sure I tried that but it still left a null entry on the undo stack.  Now I'm doubting myself, so I'll check again.